### DISCUSSION TITLE
Discounts for educators

### DISCUSSION THREAD

**Discussion by arrazaq** [community/community#18355] [GitHub Education] [answered]
Why was the educator discount application submission not approved?

**Comment by andae** [comment, marked as answer]
Maybe you missed something, click [this link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-in-your-classroom-and-research/why-wasnt-my-application-for-an-educator-or-researcher-discount-approved)! Hope you can help

**Reply by arrazaq** [reply]
Thank you

**Comment by AftabAhmedAbro** [comment]
It takes time to approve because when I applied for the Developer kit first time it was approved in more than 20 days and second time it took few mintues

**Comment by marcosp75** [comment]
Hi there,

I am a university teacher, and I’m having trouble verifying my submission. I would appreciate an email address where I can send all the necessary information to resolve this issue.

I have already provided and confirmed my university email, and I have uploaded my digital ID from the university multiple times. Unfortunately, I’m unsure how to proceed further.

Could someone from GitHub please assist with this?

Thank you in advance for your help.

Best regards,
Marcos
